scripts/drone/steps/lib.star

```diff
@@ -775,11 +775,22 @@ def lint_frontend_step():
         "commands": [
             "yarn run prettier:check",
             "yarn run lint",
-            "yarn run i18n:compile",  # TODO: right place for this?
             "yarn run typecheck",
         ],
     }
 
+def verify_i18n_step():
+    return {
+        "name": "verify-i18n",
+        "image": build_image,
```

Comment:
I noticed that the other frontend steps have 
```
        "environment": {
            "TEST_MAX_WORKERS": "50%",
        },
```
Is this something needed here too?